src/RuleSet/Sets/SymfonySet.php

```diff
@@ -147,6 +147,10 @@ public function getRules(): array
                 ],
                 'sort_algorithm' => 'alpha',
             ],
+            'ordered_types' => [
+                'null_adjustment' => 'always_last',
+                'sort_algorithm' => 'none',
```

Comment:
can't we use any sorting here?

Reply:
regardless of decision of the Sf, we can at least do it for us.
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7361

Reply:
Since they use 
```
'phpdoc_types_order' => [
     'null_adjustment' => 'always_last',
     'sort_algorithm' => 'none',
],
```
I don't see why they would sort the types and not the phpdoc.

But indeed PhpCsFixerSet can use `ordered_type => true`

Reply:
let me rephrase my original question - wouldn't Symfony as community be interested to sort in both cases?

Reply:
Dunno 🤷‍♂️
I only know about their standard here: https://symfony.com/doc/current/contributing/code/standards.html